packages/core/src/services/experiments/assertRequirements.ts

```diff
@@ -23,7 +22,7 @@ export function assertEvaluationRequirements({
     const datasetLabel = datasetLabels[evaluation.uuid]
     if (spec.requiresExpectedOutput && !datasetLabel) {
       return Result.error(
-        new BadRequestError(
+        new Error(
```

Comment:
ditto

Reply:
The transaction was complaining like crazy here